examples/config-dev.yaml

```diff
@@ -17,6 +17,8 @@ staticClients:
 - id: example-app
   redirectURIs:
   - 'http://127.0.0.1:5555/callback'
+  - 'http://localhost:9000/auth/callback'
+  - 'urn:ietf:wg:oauth:2.0:oob'
```

Comment:
I don't think this actually has to be registered. It's implied by `public: true`